docs/source-pytorch/starter/installation.rst

```diff
@@ -56,6 +38,22 @@ You can also use `Conda Environments <https://docs.conda.io/projects/conda/en/la
 
 ----
 
+*****************************
+Apple Silicon (M1/M2/M3) Macs
+*****************************
+While ML related python packages are updated to work with Apple Silicon, you'll need to set 2 environment variables on install.
+
+.. code-block:: bash
+
+    # needed for M1/M2/M3
+    export GRPC_PYTHON_BUILD_SYSTEM_OPENSSL=1
+    export GRPC_PYTHON_BUILD_SYSTEM_ZLIB=1
+
+    python -m pip install -U lightning
+
+
+----
+
```

Comment:
```suggestion

In case you face difficulty with pulling the GRPC package, please follow this `thread <https://stackoverflow.com/questions/66640705/how-can-i-install-grpcio-on-an-apple-m1-silicon-laptop>`_


----

```